src/pageObjects/welcomePage/WelcomePage.js

```diff
@@ -0,0 +1,22 @@
+const { expect } = require('@playwright/test')
+import BasePage from '../BasePage'
+
+export default class WelcomePage extends BasePage {
+    constructor(page) {
+        super(page, '/', page.locator('.guest-layout'))
+        this.signUpButton = this._page.locator('button', {hasText: 'Sign Up'})
+        this.registrationPopup = this._page.locator('.modal-dialog.modal-dialog-centered')
+    }
+    
+    
+    async getRegistrationPopup(){
+        await this.signUpButton.click()
+        await expect(this.registrationPopup).toBeVisible()
```

Comment:
remove this expect and return component RegistrationPopup

Reply:
fixed